README.md

```diff
@@ -389,6 +389,49 @@ describe App do
 end
 ```
 
+In order to stub the response to actual queries, [dump the schema into a JSON file](#schema-storing-and-loading-on-disk) and specify it via schema_path as follows.
+
+```ruby
+describe App do
+  let(:url) { 'http://graph.biz/graphql' }
```

Comment:
I think you need `include Rack::Test::Methods` as well.

Reply:
`include Rack::Test::Methods` are not necessary when using rspec and webmock. That's the reason the test work in the suite. Incluiding it will lead to confusion as users may think that is required in order to work.